src/utils/trust/details.mapper.ts

```diff
@@ -51,25 +64,29 @@ const mapDetailToPage = (
     ceasedDateYear: trustData.ceased_date_year,
     hasAllInfo: unableToObtainAllTrustInfo,
     beneficialOwnersIds: trustBoIds,
+    stillInvolved: stillInvolvedInOverseasEntity,
   };
 };
 
 //  to session mapping
 const mapDetailToSession = (
   formData: Page.TrustDetailsForm,
-  isTrustToBeCeased: boolean
+  isTrustToBeCeased: boolean,
+  isTrustStillInvolved: boolean
 ): Trust => {
   const data = formData;
+  const setCeasedDate: boolean = isTrustToBeCeased || !isTrustStillInvolved;
 
   return {
     trust_id: data.trustId,
     trust_name: data.name,
     creation_date_day: data.createdDateDay,
     creation_date_month: data.createdDateMonth,
     creation_date_year: data.createdDateYear,
-    ceased_date_day: isTrustToBeCeased ? data.ceasedDateDay : undefined,
-    ceased_date_month: isTrustToBeCeased ? data.ceasedDateMonth : undefined,
-    ceased_date_year: isTrustToBeCeased ? data.ceasedDateYear : undefined,
+    ceased_date_day: setCeasedDate ? data.ceasedDateDay : undefined,
+    ceased_date_month: setCeasedDate ? data.ceasedDateMonth : undefined,
+    ceased_date_year: setCeasedDate ? data.ceasedDateYear : undefined,
+    trust_still_involved_in_overseas_entity: isTrustStillInvolved ? "Yes" : "No",
```

Comment:
the model type for trust_still_involved_in_overseas_entity is tring to allow null/undefined, but this line looks like null/undefined can't happen

Reply:
👍 again, related to the 'blue screen' scenario. Investigating.

Reply:
Fixed this. See above comment.